documentation/asciidoc/topics/proc_configuring_sites_automatically.adoc

```diff
@@ -4,6 +4,16 @@
 [role="_abstract"]
 Configure {ispn_operator} to establish cross-site views with {brandname} clusters.
 
+[NOTE]
```

Comment:
Sorry, I told you the wrong file before :scream:

I think `documentation/asciidoc/topics/proc_securing_cross_site_connections.adoc` is the one we need.